templates/Containerfile

```diff
@@ -52,40 +52,40 @@ ARG IMAGE_NAME="{{ recipe.name }}"
 ARG BASE_IMAGE="{{ recipe.base_image }}"
 
 {%- for module in recipe.modules_ext.modules %}
-  {%- if let Some(type) = module.module_type %}
-    {%- if type == "containerfile" %}
-      {%- if let Some(containerfiles) = self::get_containerfile_list(module) %}
-        {%- for c in containerfiles %}
+{%- if let Some(type) = module.module_type %}
+{%- if type == "containerfile" %}
+{%- if let Some(containerfiles) = self::get_containerfile_list(module) %}
+{%- for c in containerfiles %}
 {{ self::print_containerfile(c) }}
```

Comment:
Please stop removing the indents here. It's hard to follow the scoping without it

Reply:
Must be the linter or formatted because I'm not touching the file 👍

Reply:
Yeah the PR I'll be doing to seperate out all this stuff, I'm going to give this file the `.j2` file extension so it has some sort of proper file highlighting